services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -137,12 +137,12 @@ export class NoticeOfWorkTable extends Component {
     {
       title: "NoW No.",
       dataIndex: "nowNum",
-      sortField: "tracking_number",
+      sortField: "now_number",
       render: (text, record) => (
-        <Link to={router.NOTICE_OF_WORK_INITIAL_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
+        <Link to={router.VIEW_NOTICE_OF_WORK_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
```

Comment:
Is there a view button and a link on the now_number that route to the same place? if so, should we remove this link and just have the view button?

Reply:
It crossed my mind but I see no harm in leaving it either. I like having the links in general.